opba-banking-rest-impl/src/main/java/de/adorsys/opba/tppbankingapi/controller/TppBankSearchController.java

```diff
@@ -51,7 +52,8 @@
         if (onlyActive == null) {
             onlyActive = true;
         }
-        List<BankDescriptor> banks = bankService.getBanks(keyword, start, max, onlyActive);
+
+        List<BankDescriptor> banks = keyword.isBlank() ? Collections.emptyList() : bankService.getBanks(keyword, start, max, onlyActive);
```

Comment:
The result of bank search with empty string should be the same as with ` ` space

Reply:
fixed